<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QCheckBox" name="checkBox1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Check1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Check2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Check3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(0, 170, 127);</string>
   </property>
   <property name="text">
    <string>机械臂</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(0, 170, 127);</string>
   </property>
   <property name="text">
    <string>姿轨控</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(0, 170, 127);</string>
   </property>
   <property name="text">
    <string>无线电</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIP">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPort">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditIP">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPort">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
